spec/controllers/concerns/client_sortable_spec.rb

```diff
@@ -159,6 +172,13 @@
           expect(subject.has_search_and_sort_params?).to eq true
         end
       end
+
+      context "vita_partner_id" do
+        let(:params) { { vita_partner_id: 1 } }
+        it "returns true" do
+          expect(subject.has_search_and_sort_params?).to eq true
```

Comment:
Are we using `.has_search_and_sort_params?` outside of the spec? I couldn't find any usages. Was it added for testing purposes?

Reply:
it was being used on the client search. I guess its not being used anymore since that code was removed,